Files/MultilingualResources/Files.de-DE.xlf

```diff
@@ -1139,6 +1139,22 @@
           <source>Invalid item</source>
           <target state="new">Invalid item</target>
         </trans-unit>
+        <trans-unit id="ShareDialogFailMessage" translate="yes" xml:space="preserve">
+          <source>There's nothing to share right now</source>
+          <target state="new">There's nothing to share right now</target>
+        </trans-unit>
+        <trans-unit id="ShareDialogMultipleItemsDescription" translate="yes" xml:space="preserve">
+          <source>The items you've selected will be shared</source>
+          <target state="new">The items you've selected will be shared</target>
+        </trans-unit>
+        <trans-unit id="ShareDialogSingleItemDescription" translate="yes" xml:space="preserve">
+          <source>The selected item will be shared</source>
+          <target state="new">The selected item will be shared</target>
+        </trans-unit>
+        <trans-unit id="ShareDialogTitle" translate="yes" xml:space="preserve">
+          <source>Sharing {0}</source>
```

Comment:
It is better to use `Sharing {0} {1}` and `string.Format(ResourceController.GetTranslation("ShareDialogTitle"), items.Count, ResourceController.GetTranslation("ItemsCount.Text"))`

Reply:
Done

Reply:
Please, update translation files too.